en/asgardeo/docs/guides/users/manage-users.md

```diff
@@ -210,6 +210,11 @@ To lock a user account:
 !!! note
```

Comment:
Please adjust all the occurrences to match the following :

## Lock a user account

When you lock a user's account, the user will no longer be able to access any applications or the My Account portal.

To lock a user account:

1. On the {{product_name}} Console, go to **User Management** > **Users** and select the user.
2. In the **Danger Zone**, turn the **Lock user** toggle on to lock the user's profile. Turn it off to unlock it.
  
    ![Lock user]({{base_path}}/assets/img/guides/users/lock-user.png){: width="600" style="display: block; margin: 0; border: 0.3px solid lightgrey;"}

3. Select the checkbox to confirm your action.
4. Click **Confirm**.

!!! note
    When a user account is locked, the reason for the account lock will be displayed in the user's profile.

    ![Account lock reason]({{base_path}}/assets/img/guides/users/lock-user-profile.png){: width="600" style="display: block; margin: 0; border: 0.3px solid lightgrey;"}

Reply:
Resolved with c4009f0fab2233e22fec947ae31e254281ee92d7